### DISCUSSION TITLE
UNO Extensions Template Hot Reload Issue

### DISCUSSION THREAD

**Discussion by jpgarza93** [unoplatform/uno#10488] [General]
Hello, I have created a new project using `dotnet new unoapp-extensions` , it runs fine, but off the bat, Hot Reload is not working for the WinUI project

**Comment by jeromelaban** [comment]
For the WinUI hot reload to work, you need to make sure that the top-left project selector in the XAML editor is on the `.Windows` project.

**Reply by jpgarza93** [reply]
I have that selected it and still no Hot Reload or XAML Live Preview. I am running Windows Debug x64 in VS 2022 (version 17.3.6)

**Reply by jeromelaban** [reply]
Thanks. If you create a WinAppSDK app without the uno templates, does it work for you?

**Reply by jpgarza93** [reply]
Created a clean/new WinUI 3 project and a UNO Platform. Hot Reload and XAML Live Preview worked in both. Just not working on a UNO platform extensions template project

![image](https://user-images.githubusercontent.com/79015116/203114856-5bbf19da-179f-4f4a-bb4b-a40c4f87108e.png)
![image](https://user-images.githubusercontent.com/79015116/203116502-be42492d-217a-4977-8e31-3921a2731a14.png)

**Reply by jeromelaban** [reply]
Thanks, this helps. @nickrandolph have you noticed this as well ?

**Reply by jpgarza93** [reply]
While Hot Reload or XAML Live Preview does not work on the WinUI project they do work on the Skia.WPF project

**Reply by nickrandolph** [reply]
> Thanks, this helps. @nickrandolph have you noticed this as well ?

We'll test this with the latest dev release of the extensions template and let you know
